treefmt.toml

```
@@ -0,0 +1,16 @@
+[formatter.nix]
+command = "nixfmt"
+includes = ["*.nix"]
+
+[formatter.ocaml]
+command = "dune"
+options = ["build", "@fmt", "--auto-promote"]
+includes = ["*.ml", "dune"]
+
+[formatter.javascript]
+command = "prettier"
+includes = ["*.js"]
+
+[formatter.sh]
+command = "shellcheck"
+includes = ["*.sh"]
```

```suggestion

[formatter.sh]
command = "shellcheck"
includes = ["*.sh"]
```
I don't think shellcheck does formatting